render-test/render_test.cpp

```diff
@@ -177,9 +178,11 @@ int runRenderTests(int argc, char** argv, std::function<void()> testStatus) {
             }
         }
 
-        bool errored = !metadata.errorMessage.empty() || !runner.run(metadata);
-        bool passed =
-            !errored && (!metadata.outputsImage || !metadata.diff.empty()) && metadata.difference <= metadata.allowed;
+        auto results = runner.run(metadata);
```

Comment:
This changes previous behavior.  I think that if metadata parsing has failed, test should not be run, e.g.,
```diff
-        auto results = runner.run(metadata);
-
-        bool errored = !metadata.errorMessage.empty() || !(results.probeFailed || results.renderFailed);
+        TestRunner::TestResults results{};
+        bool errored = !metadata.errorMessage.empty();
+        if (!errored) {
+            results = runner.run(metadata);
+            errored = !(results.probeFailed || results.renderFailed);
+        }
```